content/en/synthetics/browser_tests/advanced_options.md

```diff
@@ -43,9 +43,17 @@ In some cases, such as in the event of a pop-up, you may want to make some steps
 
 {{< img src="synthetics/browser_tests/advanced_options/timeout.png" alt="Timeout" style="width:25%">}}
 
+## Prevent screenshot capture
+
+You can prevent a step screenshot from being captured at test execution. This can be helpful to ensure no sensitive data gets featured in your test results. It however needs to be use mindfully as it can make failure troubleshooting way more challenging. You can read more about our security recommendations [here][3].
```

Comment:
```suggestion
You can prevent a step screenshot from being captured at test execution. This can be helpful to ensure no sensitive data gets featured in your test results. It however needs to be used mindfully as it can make failure troubleshooting way more challenging. You can read more about our security recommendations [here][3].
```